python/paddle/fluid/dygraph/dygraph_to_static/loop_transformer.py

```diff
@@ -217,21 +253,30 @@ def while_loop_body(self.x, y) # `self.x` is illegal.
         if attr_full_name.startswith("self."):
             return
         self.current_seen_vars.add(node)
+
+        write_context = {
+            type(gast.Store()), type(gast.AugStore()), type(gast.Del())
+        }
```

Comment:
`write_context` doesn't seem to be used in `visit_FunctionDef `.

Reply:
Thank you for reviewing carefully. I will delete it.